readme.md

```diff
@@ -75,6 +75,13 @@ Type: `Buffer`
 
 Buffer to optimize.
 
+#### keepChunks
+
+Type: `String`
```

Comment:
It would be better as an array and you can just `.join(',')` in the code.

It should also document the possible values. And could also link to http://www.libpng.org/pub/png/spec/1.2/PNG-Chunks.html to read more about the chunks.